dist/js/tcec.js

```diff
@@ -3072,12 +3072,14 @@ function updateLiveEvalDataHistory(datum, fen, container, contno)
    }
 
    /* Check if black */
-   if (datum.pv.search(/.*\.\.\..*/i) == 0)
+   if(regexBlackMove.test(datum.pv)) {
    {
       if (!isNaN(score))
       {
          /* Invert the score */ 	  
+         plog ("TCECROCKS: Score before conversion is :" + score);	      
```

Comment:
Is this stuff intentionally in the pull request?

Reply:
Yes, so that can we filter the output, once i understand the issue i can remove it

Reply:
Are you perhaps using Chrome or Chronium based browser? If yes, then you can do it locally:
https://stackoverflow.com/questions/16494237/chrome-dev-tools-modify-javascript-and-reload
https://www.zyxware.com/articles/3549/how-to-edit-javascript-code-in-your-browser-and-see-the-changes-in-real-time